bigbluebutton-html5/imports/ui/components/tabs/TabsContainer.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import Service from './service';
+import { withModalMounter } from '/imports/ui/components/modal/service';
+import ExternalVideoService from '/imports/ui/components/external-video-player/service';
+import PresentationService from '../presentation/presentation-uploader/service';
+import PresentationUploaderContainer from '/imports/ui/components/presentation/presentation-uploader/container';
+import { UPLOAD_FILE_TYPES } from '/imports/api/presentations/constants';
+
+import TabsView from './TabsView';
+
+const TABS = [
+  {
+    id: 0,
+    fileType: UPLOAD_FILE_TYPES.PPT,
+    icon: 'ppt',
+  },
+  {
+    id: 1,
+    fileType: UPLOAD_FILE_TYPES.PDF,
+    icon: 'pdf',
+  },
+  {
+    id: 2,
+    fileType: UPLOAD_FILE_TYPES.PDF,
+    icon: 'video',
+  },
+  {
+    id: 3,
+    fileType: UPLOAD_FILE_TYPES.PDF,
+    icon: 'www',
+  },
+];
+
+
+const TabsContainer = ({ mountModal, ...props }) => {
+  const [tabIndex, setTabIndex] = useState(0);
+  const [selectedOption, setSelectedOption] = useState(null);
+
+  const SELECTED_OPTION_BY_TYPE = {
+    [UPLOAD_FILE_TYPES.PPT]: null,
+    [UPLOAD_FILE_TYPES.PDF]: null,
+  };
```

Comment:
it should be like this

```
const [selectedOption, setSelectedOption] = useState({
   [UPLOAD_FILE_TYPES.PPT]: null,
   [UPLOAD_FILE_TYPES.PDF]: null,
});
```

on `setSelectedOption` should be

```
setSelectedOption({ [fileType]: e.value });
```

Reply:
fixed